src/main/resources/application-prod.yml

```
@@ -1,18 +1,47 @@
   datasource:
-    username: ${DB_USERNAME}
-    password: ${DB_PASSWORD}
     driver-class-name: org.postgresql.Driver
+    url: ${ secrets.RDS_URL }
+    username: ${ secrets.RDS_USERNAME }
+    password: ${ secrets.RDS_PASSWORD }
 
   jpa:
     hibernate:
-      ddl-auto: validate
+      ddl-auto: update # 초기 배포 확인용으로 update 사용.
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)